scheduler/src/cook/mesos/unscheduled.clj

```diff
@@ -150,13 +151,15 @@
   "Return the appropriate error message if a user's job is unscheduled because they're over the job launch rate limit threshold"
   [{:keys [job/user]}]
   (let [enforcing-job-launch-rate-limit? (ratelimit/enforce? ratelimit/job-launch-rate-limiter)
-        time-until-out-of-debt (ratelimit/time-until-out-of-debt-millis! ratelimit/job-launch-rate-limiter user)
-        in-debt? (not (zero? time-until-out-of-debt))
+        num-ratelimited (->> @scheduler/pool->user->number-jobs
+                             vals
+                             (map #(get % user 0))
+                             (reduce + 0))
+        being-ratelimited? (not (zero? num-ratelimited))
         {:keys [tokens-replenished-per-minute]} ratelimit/job-launch-rate-limiter]
-    (when (and enforcing-job-launch-rate-limit? in-debt?)
-      ["You have exceeded the limit of jobs launched per minute."
-       {:max-jobs-per-minute tokens-replenished-per-minute
-        :seconds-until-can-launch (-> time-until-out-of-debt (/ 1000.0) Math/ceil int)}])))
+    (when (and enforcing-job-launch-rate-limit? being-ratelimited?)
+      ["You are currently limited by the maximum jobs per minute."
```

Comment:
"You are currently rate limited on how many jobs you launch per minute"?

Reply:
Changed.